src/main/java/paperai/pdfextract/PDFExtractor.java

```diff
@@ -74,6 +76,7 @@ static void processFile(File file) {
                 ext.write();
             }
             w.close();
+            doc.close();
```

Comment:
I guess the "close" should be in the finally block here as well.

Btw. how about using a "try with resources"?

Reply:
I thougth about that and wanted to keep modifications as small as possible.
However it is probably better to change this. :+1: